### PR TITLE
add use binary section

### PR DESCRIPTION
**Pull request by oxarbitrage**
After a build there is no glue on where the binaries created are located. 

````
alfredo@alfredo-System-Product-Name:~/bitshares-gitian$ ./run-gitian -b 3.1.0 -m 8192
Submodule path 'docs': checked out '0271e725130383b1af3555a17efcc939b48c39e5'
Submodule path 'libraries/fc': checked out 'af572ba7d016136883c31daaf054e03e5f3695a1'
Submodule path 'libraries/fc/vendor/editline': checked out 'fbb1f8800adbb70264fa3893dc221f524e25708c'
Submodule path 'libraries/fc/vendor/secp256k1-zkp': checked out 'bd067945ead3b514fba884abd0de95fc4b5db9ae'
Submodule path 'libraries/fc/vendor/websocketpp': checked out '4543c5f9860333e9c73552362ae74239f98e80fe'
--- Building for bionic amd64 ---
Stopping target if it is up
Making a new image copy
Starting target
Checking if target is up.
Preparing build environment
Updating apt-get repository (log in var/install.log)
Installing additional packages (log in var/install.log)
Upgrading system, may take a while (log in var/install.log)
Creating package manifest
Creating build script (var/build-script)
Running build script (log in var/build.log)
Grabbing results from target
Grabbing cache from target
Generating report
c51d87f37ac1f03624b56bb97214f35192186bc5ef36e5e8af5cf771da8fdfe6  bitshares-3.1.0-linux-amd64-bin.tar.bz2
ea2be11b6622a59f6ab0258215b6904d8368d348907d4fb56b5c71ca540483ee  bithares-core-linux-res.yml
Done.
```

This section should save the users some minutes searching for it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,3 +104,17 @@ This will create a subdirectory under `signatures`. If you want to contribute, p
 Example: verify version 3.1.0:
 
 `run-gitian -v 3.1.0`
+
+### Use binary
+
+A `bz2` file(`bitshares-3.1.0-linux-amd64-bin.tar.bz2`) will be genrated at `bitshares-gitian/vendor/gitian-builder/build/out/`  after any `./run-gitan -b` is executed.
```
Comment (pmconrad) [inline README.md]: Typo: gen*e*rated

File: README.md [hunk 2]
```diff
@@ -104,3 +104,17 @@ This will create a subdirectory under `signatures`. If you want to contribute, p
...
+
+Example: use built witness_node binary: 
+
+```
+cd vendor/gitian-builder/build/out/
+bunzip2 bitshares-3.1.0-linux-amd64-bin.tar.bz2
+tar xvf vendor/gitian-builder/build/out/
+tar bitshares-3.1.0-linux-amd64-bin.tar
```
Comment (pmconrad) [inline README.md]: Please replace the previous 3 lines with
```
tar xvfj bitshares-3.1.0-linux-amd64-bin.tar.bz2
```

Comment (oxarbitrage) [reply README.md]: my apologies, the second line of this was wrong, however changed to 1 line as suggested.

**Comment by oxarbitrage** [comment]
fixed conflict.

**Pull request merged**